k8s/overlays/azure-commons/cgmes-assembling-job-config.yml

```diff
@@ -10,7 +10,3 @@ case-server:
 cgmes-boundary-server:
   url: http://cgmes-boundary-server/
 
-database:
-  url: jdbc:postgresql://10.0.0.5:5432/cgmes_assembling
```

Comment:
Where does this config has moved ?

Reply:
database.properties the same file as the other things